### PR TITLE
Add config for Bazel CI

### PR DESCRIPTION
**Pull request by Yannic**
This PR adds a simple config to test parts of gRPC on Ubuntu 16.04.
We will add more platforms (macOS, Windows, RBE, ...) when the build is
green.

See #19171 

@yashykt

//cc @keith

### CODE REVIEW

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+# TODO(yannic): Ideally, we should also enable buildifier and all platforms should test `//...`.
```
Comment (jtattermusch) [inline .bazelci/presubmit.yml]: please add more info about what is the purpose of this file and who to reach out to with questions.
Can you also add a link to the test results? (Once known)

Comment (Yannic) [reply .bazelci/presubmit.yml]: Thanks, done.

I'll send a follow-up with the link when it's known (Filed https://github.com/bazelbuild/continuous-integration/issues/857 to create the pipeline)

File: .bazelci/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Configuration file for Bazel CI [1].
+#
+# Also testing on Bazel CI in addition of our normal CI workflow
+# ensures that gRPC is tested agains Bazel@HEAD and stays compatible
```
Comment (jtattermusch) [inline .bazelci/presubmit.yml]: nit: s/agains/against/

Comment (Yannic) [reply .bazelci/presubmit.yml]: Sorry (typing is hard)

**Comment by Yannic** [comment]
@jtattermusch Any news on this?

**Pull request merged**